lib/api/build.rs

```diff
@@ -111,14 +111,14 @@ fn configure_validation(builder: Builder) -> Builder {
             ("VectorParams.quantization_config", ""),
             ("QuantizationConfig.quantization", ""),
             ("ScalarQuantization.quantile", "custom = \"crate::grpc::validate::validate_f32_range_min_0_5_max_1\""),
-            ("ProductQuantization.bucket_size", "custom = \"crate::grpc::validate::validate_required_u64_range_min_1\""),
             ("VectorParamsMap.map", ""),
         ], &[
             "ListCollectionsRequest",
             "CollectionParamsDiff",
             "ListAliasesRequest",
             "CollectionClusterInfoRequest",
             "UpdateCollectionClusterSetupRequest",
+            "ProductQuantization",
```

Comment:
I don't think we have to validate this anymore, because deserialization forces it to be one of the enum variants.

That would mean we can simply remove this line.

Reply:
Want to keep because of this line:
https://github.com/qdrant/qdrant/blob/product-quantization/lib/api/src/grpc/validate.rs#L87

Reply:
Makes total sense. Missed that!